environment.yml

```
@@ -36,6 +36,7 @@ dependencies:
   - rfc3339-validator >=0.1  # for python-jsonschema date-time format validation
   - rioxarray >=0.11
   - s3fs >=2021.6
+  - adlfs >=2023.1 # for azure blob filesystem
```

```suggestion
  - adlfs >=2023.1  # for azure blob filesystem
```

PEP8 wants 2 spaces between code and comment (we apply it for YAML as well)